### PR TITLE
fix Tooltip opacity.

### PR DESCRIPTION
**Pull request by karthik-sista**

### CODE REVIEW

File: libs/core/scss/_custom.scss [hunk 1]
```diff
@@ -84,6 +84,13 @@ $spacers: (
   5: ($spacer * 3)
 );
 
+//
+//Opacity
+//
+
+$default-opacity: 1.0;
```
Comment (eckdanny-osi) [inline libs/core/scss/_custom.scss]: use the `// Tooltips` section and set `$tooltip-opacity` instead. (Default value: `9 !default;`)

Comment (eckdanny-osi) [reply libs/core/scss/_custom.scss]: ^ @karthik-sista , would you say root cause is 📘 [`CL-252`](https://osi-cwds.atlassian.net/browse/CL-252)?

File: libs/core/scss/_custom.scss [hunk 2]
```diff
@@ -212,6 +219,7 @@ $input-focus-border-color:              $accent;
 // Tooltips
 
 $tooltip-bg: $info;
+$tooltip-opacity: $default-opacity;
```
Comment (eckdanny-osi) [inline libs/core/scss/_custom.scss]:
```suggestion
$tooltip-opacity: 1.0;
```

File: libs/core/scss/_custom.scss (hunk 1 above)
Comment (karthik-sista) [reply libs/core/scss/_custom.scss]: Kind of. Yeah

File: libs/core/scss/_custom.scss [hunk 3]
```diff
@@ -84,6 +84,7 @@ $spacers: (
   5: ($spacer * 3)
 );
 
+
```
Comment (eckdanny-osi) [inline libs/core/scss/_custom.scss]:
```suggestion

```

**Pull request merged**